spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -500,6 +624,305 @@ command is:kubectl logs test-log-0-ysiz3
 dataflow:>stream destroy --name test
 ```
 
+=== Deploying Streams using Skipper
+We will proceed with the assumption that Spring Cloud Data Flow, <<getting-started.adoc#spring-cloud-skipper-integration, Spring Cloud Skipper>>, RDBMS, and desired messaging 
+middleware is up and running in minikube.
+
+[source,console,options=nowrap]
+----
+$ kubectl get all
+NAME                              READY     STATUS    RESTARTS   AGE
+po/mysql-777890292-z0dsw          1/1       Running   0          38m
+po/rabbitmq-317767540-2qzrr       1/1       Running   0          38m
+po/redis-4054078334-37m0l         1/1       Running   0          38m
+po/scdf-server-2734071167-bjd3g   1/1       Running   0          48s
+po/skipper-2408247821-50z31       1/1       Running   0          3m
+
+...
+...
+----
+
+Verify the available platforms in Skipper.
+
+[source,console,options=nowrap]
+----
+skipper:>platform list
+╔════════╤══════════╤══════════════════════════════════════════════════════════════════════════════════════════════════════════════╗
+║  Name  │   Type   │                                                 Description                                                  ║
+╠════════╪══════════╪══════════════════════════════════════════════════════════════════════════════════════════════════════════════╣
+║default │local     │ShutdownTimeout = [30], EnvVarsToInherit = [TMP,LANG,LANGUAGE,LC_.*,PATH], JavaCmd =                          ║
```

Comment:
By disabling the `local` platform, we could avoid seeing this when using SCDF + Kubernetes (or CF).